src/main/assets/js/fact-tabs/fact-tabs.js

```diff
@@ -0,0 +1,280 @@
+  //this.changingHash = true
+  window.location.hash = this.getHref($tab).slice(1);
+  $panel.id = id;
+};
+
+FactTabs.prototype.onTabKeydown = function (e) {
+  switch (e.keyCode) {
+    case this.keys.left:
+    case this.keys.up:
+      this.activatePreviousTab();
+      e.preventDefault();
+      break;
+    case this.keys.right:
+    case this.keys.down:
+      this.activateNextTab();
+      e.preventDefault();
+      break;
+  }
+};
+
+FactTabs.prototype.activateNextTab = function () {
+  var currentTab = this.getCurrentTab();
+  var nextTabListItem = currentTab.parentNode.nextElementSibling;
+  if (nextTabListItem) {
+    var nextTab = nextTabListItem.querySelector('.fact-tabs-tab');
+  }
+  if (nextTab) {
+    this.hideTab(currentTab);
+    this.showTab(nextTab);
+    nextTab.focus();
+    this.createHistoryEntry(nextTab);
+  }
+};
+
+FactTabs.prototype.activatePreviousTab = function () {
+  var currentTab = this.getCurrentTab();
+  var previousTabListItem = currentTab.parentNode.previousElementSibling;
+  if (previousTabListItem) {
+    var previousTab = previousTabListItem.querySelector('.fact-tabs-tab');
+  }
+  if (previousTab) {
+    this.hideTab(currentTab);
+    this.showTab(previousTab);
+    previousTab.focus();
+    this.createHistoryEntry(previousTab);
+  }
+};
+
+FactTabs.prototype.getPanel = function ($tab) {
+  var $panel = this.$module.querySelector(this.getHref($tab));
+  return $panel;
+};
+
+FactTabs.prototype.showPanel = function ($tab) {
+  var $panel = this.getPanel($tab);
+  $panel.classList.remove(this.jsHiddenClass);
+};
+
+FactTabs.prototype.hidePanel = function (tab) {
+  var $panel = this.getPanel(tab);
+  $panel.classList.add(this.jsHiddenClass);
+};
+
+FactTabs.prototype.unhighlightTab = function ($tab) {
+  $tab.setAttribute('aria-selected', 'false');
+  $tab.parentNode.classList.remove('fact-tabs-list-item--selected');
+  $tab.setAttribute('tabindex', '-1');
+};
+
+FactTabs.prototype.highlightTab = function ($tab) {
+  $tab.setAttribute('aria-selected', 'true');
+  $tab.parentNode.classList.add('fact-tabs-list-item--selected');
+  $tab.setAttribute('tabindex', '0');
+};
+
+FactTabs.prototype.getCurrentTab = function () {
+  return this.$module.querySelector('.fact-tabs-list-item--selected .fact-tabs-tab');
+};
+
+// this is because IE doesn't always return the actual value but a relative full path
+// should be a utility function most prob
+// http://labs.thesedays.com/blog/2010/01/08/getting-the-href-value-with-jquery-in-ie/
+FactTabs.prototype.getHref = function ($tab) {
+  var href = $tab.getAttribute('href');
+  var hash = href.slice(href.indexOf('#'), href.length);
+  return hash;
+};
+
+FactTabs.prototype.getSelectedTab = function() {
+  var text = this.$module.querySelector('.fact-tabs-title').innerText;
+  return this.$module.querySelector('.fact-tabs-tab').innerText = text;
+};
+
+FactTabs.prototype.setSelected = function(string){
+  var selectedTab = this.$module.querySelector('.fact-tabs-title');
```

Comment:
When you select the tab in the list, the list/dropdown doesn't disappear; you have to move the mouse away. I wonder if it would be more user-friendly for the list/dropdown to close.

Reply:
would need to change it from hover on to a click event to achieve that